glossary.yml

```
@@ -8825,6 +8825,16 @@
     def: >
       A test that exercises one function or feature of a piece of software and
       produces [pass](#pass_test), [fail](#fail_test), or [error](#error_test).
+      
+- slug: unix
+  ref:
+    - operating_system
+  en:
+    term: "UNIX"
+    def: >
+      UNIX is a family of [operating systems](#operating-system) developed during
```

The slug for "operating system" has an underscore, not a hyphen (because hyphens are special in YAML)

```suggestion
      UNIX is a family of [operating systems](#operating_system) developed during
```

---

When this is changed, the tests will pass and we can merge